src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgres.java

```diff
@@ -475,10 +560,10 @@ private File prepareBinaries(PgBinaryResolver pgBinaryResolver) {
                             try {
                                 Preconditions.checkState(!pgDirExists.exists(), "unpack lock acquired but .exists file is present.");
                                 LOG.info("Extracting Postgres...");
-                                system("tar", "-x", "-f", pgTbz.getPath(), "-C", pgDir.getPath());
+                                extractTbz(pgTbz.getPath(), pgDir.getPath());
                                 Files.touch(pgDirExists);
-                            } finally {
-                                Preconditions.checkState(unpackLockFile.delete(), "could not remove lock file %s", unpackLockFile.getAbsolutePath());
+                            } catch (Exception e) {
+                                LOG.error(e.getMessage());
```

Comment:
Calling `e.getMessage()` destroys the stack trace.
Prefer `LOG.error("while unpacking postgres", e)` which presents the stack trace.